lisa/tools/echo.py

```diff
@@ -18,6 +18,19 @@ def _windows_tool(cls) -> Optional[Type[Tool]]:
     def _check_exists(self) -> bool:
         return True
 
+    def write_to_file(
+        self,
+        val: str,
+        file_path: str,
+        sudo: bool = False,
+        super_user: bool = False,
+    ) -> None:
+        # Run `echo <val> > <file_path>`
+        output = self.run(
+            f"{val} > {file_path}", sudo=sudo, super_user=super_user
```

Comment:
because it may need to rewrite multiple times, it needs to force rerun every time.

Reply:
"'{val}' > {file_path}"?